ecommerce/src/main/kotlin/io/hhplus/ecommerce/product/application/ProductStatisticsSyncScheduler.kt

```
@@ -0,0 +1,127 @@
+                val productId = extractProductId(key) ?: return@forEach
+                val count = productStatisticsCacheService.getAndClearSalesCount(productId)
+
+                if (count > 0) {
+                    syncSalesCount(productId, count)
+                    syncedSalesCount++
+                }
+            }
+
+            logger.info("[ProductStatistics] Redis → DB 동기화 완료: 조회수 ${syncedViewCount}건, 판매량 ${syncedSalesCount}건")
+
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)